src/pyuvdata/uvflag/uvflag.py

```diff
@@ -2092,6 +2091,26 @@ def __add__(
                 this.telescope.antenna_diameters = temp_ant_diameters[unique_inds]
             else:
                 this.telescope.antenna_diameters = None
+            if (
+                this.telescope.feed_array is not None
+                and other.telescope.feed_array is not None
+            ):
+                temp_feed_array = np.concatenate(
+                    [this.telescope.feed_array, other.telescope.feed_array], axis=0
+                )
+                this.telescope.feed_array = temp_feed_array[unique_inds]
+            else:
+                this.telescope.feed_array = None
+            if (
+                this.telescope.feed_angle is not None
+                and other.telescope.feed_angle is not None
+            ):
+                temp_feed_angle = np.concatenate(
+                    [this.telescope.feed_angle, other.telescope.feed_angle], axis=0
+                )
+                this.telescope.feed_angle = temp_feed_angle[unique_inds]
+            else:
+                this.telescope.feed_angle = None
```

Comment:
could this be replaced by a call to the telescope add function like you're doing in the UVData add?